litgpt/chat/base.py

```diff
@@ -163,14 +174,18 @@ def main(
     )
     stop_tokens = prompt_style.stop_tokens(tokenizer)
 
-    print(f"Now chatting with {config.name}.\nTo exit, press 'Enter' on an empty prompt.\n")
+    print(
+        f"Now chatting with {config.name}.\nTo exit, type 'exit'"
+        " or 'quit' and press 'Enter'.\nAfter entering your prompt,"
+        " hit 'Enter' once to start writing on a new line. Hit 'Enter'"
+        " twice to submit the prompt.")
```

Comment:
I find it a bit arbitrary to support 1 line break but not 2, or 2 but not 3, ...

It would be worth to instead go for a more complete solution directly.

For instance, https://github.com/simonw/llm makes you enclose your multi-line test in a 

```
!multi
your text
...
...
!end
```

block.


Another alternative would be to find out if we register Ctrl+Enter to send the prompt so that Enter can be a simple line break.

There might be other solutions